### ISSUE TITLE
数据库类型postgres，主键类型string，insert操作返回 0，LastInsertId is not supported by this driver

### ISSUE THREAD

**Issue by YungRun** [astaxie/beego#3830]
Please answer these questions before submitting your issue. Thanks!

1. What version of Go and beego are you using (`bee version`)?
```
go version go1.12.7 linux/amd64
github.com/astaxie/beego v1.12.0
github.com/lib/pq v1.2.0
```

2. What operating system and processor architecture are you using (`go env`)?

```
GOARCH="amd64"
GOBIN="/usr/local/go/bin:/home/summer/go/bin"
GOCACHE="/home/summer/.cache/go-build"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOOS="linux"
GOPATH="/home/summer/go"
GOPROXY="https://goproxy.io"
GORACE=""
GOROOT="/usr/local/go"
GOTMPDIR=""
GOTOOLDIR="/usr/local/go/pkg/tool/linux_amd64"
GCCGO="gccgo"
CC="gcc"
CXX="g++"
CGO_ENABLED="1"
GOMOD="/home/summer/workspace/gomod/pgteset/go.mod"
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build837144964=/tmp/go-build -gno-record-gcc-switches"

```
3. What did you do?
```golang
package main

import (
	"fmt"
	"github.com/astaxie/beego/orm"
	_ "github.com/lib/pq"
)

type User struct {
	Id      string   `orm:"column(id);size(64);pk"`
	Name    string
	Profile *Profile `orm:"rel(one)"`      // OneToOne relation
	Post    []*Post  `orm:"reverse(many)"` // 设置一对多的反向关系
}

type Profile struct {
	Id   string `orm:"column(id);size(64);pk"`
	Age  int16
	User *User `orm:"reverse(one)"` // 设置一对一反向关系(可选)
}

type Post struct {
	Id    string  `orm:"column(id);size(64);pk"`
	Title string
	User  *User  `orm:"rel(fk)"` //设置一对多关系
	Tags  []*Tag `orm:"rel(m2m)"`
}

type Tag struct {
	Id    string  `orm:"column(id);size(64);pk"`
	Name  string
	Posts []*Post `orm:"reverse(many)"` //设置多对多反向关系
}

func init() {

	orm.RegisterDataBase("default", "postgres", "postgresql://application:123456@127.0.0.1:5432/pgtest?sslmode=disable")
	orm.RegisterModel(new(User), new(Post), new(Profile), new(Tag))
	name := "default"
	force := true
	verbose := true
	err := orm.RunSyncdb(name, force, verbose)
	fmt.Println(err)
}

func main() {
	o := orm.NewOrm()
	o.Using("default") // 默认使用 default，你可以指定为其他数据库

	profile := new(Profile)
	profile.Age, profile.Id = 30, "1"

	user := new(User)
	user.Profile = profile
	user.Name, user.Id = "slene", "1"

	fmt.Println(o.Insert(profile))
	fmt.Println(o.Insert(user))
}

```

4. What did you expect to see?
err == nil
5. What did you see instead?
0 LastInsertId is not supported by this driver
table与数据添加成功，返回结果错误

**Comment by jianzhiyao** [comment]
当前不支持 string pk，已在此 https://github.com/astaxie/beego/pull/4150 进行return error处理

**Comment by jianzhiyao** [comment]
关于支持string pk事宜，会在此 https://github.com/astaxie/beego/issues/3876 继续跟进

**Comment by jianzhiyao** [comment]
https://github.com/astaxie/beego/pull/4150 已合并到 2.0 分支，先把此pr关闭，感谢反馈

**Comment by jianzhiyao** [comment]
All relative code is merged into develop-2.0. Beego v2.0 will be published soon.

If operation ``insert`` is successful but there is error when that operation return last inserted id.
You should handler ``ErrLastInsertIdUnavailable`` like below 

```go
_,err := o.Insert(profile)
if err == ErrLastInsertIdUnavailable {
    //do something
}
```
